src/sage/geometry/polyhedron/parent.py

```diff
@@ -400,10 +400,10 @@ def some_elements(self):
         points = []
         R = self.base_ring()
         for i in range(self.ambient_dim() + 5):
-            points.append([R(i*j^2) for j in range(self.ambient_dim())])
+            points.append([R(i * j ^ 2) for j in range(self.ambient_dim())])
```

Comment:
This spacing is a bit much for my taste. Wouldn't `i * j^2` conform to the spec too?

Reply:
I am afraid not. Also, I am not sure if XOR is really the intended thing here. changing it to ` **`  breaks some doctests though

Reply:
Oh, right, this is XOR there. So spacing that agrees with precedence would be `i*j ^ 2`

Reply:
(This expression has been there since the very beginning; 7e02dce3d6a)

Reply:
https://peps.python.org/pep-0008/#other-recommendations
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."
```
# Correct:
[...]
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)
# Wrong:
[...]
x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```